lib/cli/src/commands/package/common/macros.rs

```diff
@@ -0,0 +1,79 @@
+macro_rules! make_pb {
+    ($self:ident, $msg:expr) => {{
+        let pb = indicatif::ProgressBar::new_spinner();
+        if $self.quiet {
```

Comment:
Instead of passing self, we should just pass `quiet`. That way it can be reused in places where there is no self.

```suggestion
    ($quiet:ident, $msg:expr) => {{
        let pb = indicatif::ProgressBar::new_spinner();
        if $quiet {
```